src/events/fetch-context/meetup.js

```diff
@@ -0,0 +1,31 @@
+const fetch = require('node-fetch')
+const logger = require('@architect/shared/logger')
+
+async function isMeetupUrl (url) {
+  return true
```

Comment:
Is there a way of detecting a Meetup URL here?

Reply:
Woops 😂 yes, I'll amend this!